app/studentApi.js

```diff
@@ -0,0 +1,66 @@
+"use strict"
+
+// dependencies
+const express = require("express");
+var router = express.Router();
+var router = require('./router.js');
+const studentDB = require("./studentDB.json");
+
+// private
+function checkStudents(studentId) {
+    return studentsDB.find(function(student){student.id === studentID;});
+}
+
+//Função de verificação do estudante
+//Função para pegar o ID do estudante
+function checkStudents(request, response) {
+    var studentID = request.query.mat;
+    if (studentID) {
+        
+        //checa ID
+        var student = students.find(function(student){
+             student.id === studentID; });
+             
+        //checa SITUACAO
+        var student = students.find(function(student){
+            student.id === studentID;
+            student.situation = checkSituation(student)}); 
+        
+        if (student) {
+            response.send(student);
+        } else {
+            response.send(student);
+        }
+
+//Função que verifica a situação do Aluno
+function checkSituation() {
+    var note = verifyNote(student);
+    if (note < 50) {
+        return 'REPROVADO';
+    }
+    else if (note >=50 && <60) {
```

Comment:
There is a syntax error here:

![image](https://cloud.githubusercontent.com/assets/5324414/21115270/b2e9785a-c097-11e6-9d08-084feeb7a8bf.png)